packages/views/accounts_header_buttons.jade

```diff
@@ -1,11 +1,18 @@
 template(name="accountsHeaderButtons")
   if currentUser
-    li
-      a(href="{{path route='profileEdit'}}") Profile
-    li.account
-      a.sign-out.navbar-btn(href="#")
-        i.fa.fa-sign-out
-        span Sign Out
+    li.dropdown
+      a.dropdown-toggle.admin-settings(href='#' data-toggle='dropdown' aria-expanded='false')
+        span Account
+        i.fa.fa-gear
```

Comment:
I think nesting "sign out" in a dropdown is relatively common (Sign-in or register is more important which is why it has a dedicated button—I doubt most users log out of applications too often.) I'll replace the gear with a down arrow. I suppose we can change this in the future as the navigation builds?